backend/routers/notifications.py

```diff
@@ -0,0 +1,45 @@
+import logging
+from firebase_admin import messaging
+from fastapi import APIRouter
+import database.notification as notification_db
+
+logger = logging.getLogger('uvicorn.error')
+logger.setLevel(logging.DEBUG)
+# from utils import auth
+router = APIRouter()
+
+
+@router.post('/save-token')
+def save_token(data: dict ):
+    try:
+        user_id = data['user_id']
+        token = data['token']
+        time_zone = data['time_zone']
+        notification_db.save_token(user_id, token, time_zone)    
+
+    except Exception:
+        raise HTTPException(status_code=400, detail='No valid data')
+    return {'status': 'success'}
+
+
+@router.post('/send-notification')
```

Comment:
I added it because I thought we could use the endpoint for some triggers coming from the app. I see it's redundant now. I would remove it.


Yes, it works. I tested it modal.